### ISSUE TITLE
Exception: 'set' on SLERT documentation

### ISSUE THREAD

**Issue by mschnitzer** [openSUSE/docmanager#90]
When I run `docmanager set -p hi=test ~/svn/slert/en/xml/*` for our SLERT documentation, then I get the following exception:

```
Traceback (most recent call last):
  File "/home/manuel/GitHub/docmanager/env/bin/docmanager", line 9, in <module>
    load_entry_point('docmanager==3.1.0', 'console_scripts', 'docmanager')()
  File "/home/manuel/GitHub/docmanager/env/lib64/python3.4/site-packages/docmanager-3.1.0-py3.4.egg/docmanager/__init__.py", line 37, in main
    a = Actions(parsecli(cliargs))
  File "/home/manuel/GitHub/docmanager/env/lib64/python3.4/site-packages/docmanager-3.1.0-py3.4.egg/docmanager/action.py", line 50, in __init__
    self.__xml = [ XmlHandler(x, self.__args.stop_on_error) for x in self.__files ]
  File "/home/manuel/GitHub/docmanager/env/lib64/python3.4/site-packages/docmanager-3.1.0-py3.4.egg/docmanager/action.py", line 50, in <listcomp>
    self.__xml = [ XmlHandler(x, self.__args.stop_on_error) for x in self.__files ]
  File "/home/manuel/GitHub/docmanager/env/lib64/python3.4/site-packages/docmanager-3.1.0-py3.4.egg/docmanager/xmlhandler.py", line 71, in __init__
    self.parse()
  File "/home/manuel/GitHub/docmanager/env/lib64/python3.4/site-packages/docmanager-3.1.0-py3.4.egg/docmanager/xmlhandler.py", line 110, in parse
    self.__tree = etree.parse(self._buffer, self.__xmlparser)
  File "lxml.etree.pyx", line 3310, in lxml.etree.parse (src/lxml/lxml.etree.c:72517)
  File "parser.pxi", line 1808, in lxml.etree._parseDocument (src/lxml/lxml.etree.c:106174)
  File "parser.pxi", line 1828, in lxml.etree._parseMemoryDocument (src/lxml/lxml.etree.c:106403)
  File "parser.pxi", line 1709, in lxml.etree._parseDoc (src/lxml/lxml.etree.c:105106)
  File "parser.pxi", line 1047, in lxml.etree._BaseParser._parseUnicodeDoc (src/lxml/lxml.etree.c:99534)
  File "parser.pxi", line 580, in lxml.etree._ParserContext._handleParseResultDoc (src/lxml/lxml.etree.c:94350)
  File "parser.pxi", line 690, in lxml.etree._handleParseResult (src/lxml/lxml.etree.c:95786)
  File "parser.pxi", line 620, in lxml.etree._raiseParseError (src/lxml/lxml.etree.c:94853)
lxml.etree.XMLSyntaxError: Sequence ']]>' not allowed in content, line 529, column 95
```

**Comment by mschnitzer** [comment]
WIP

**Comment by mschnitzer** [comment]
This is fixed in 3.2.1 -> https://github.com/openSUSE/docmanager/commit/be697cff8a324f95cf6eeec3d52765242b06685f#diff-b65de56d443789c5f05a02e826fac976R113
